libvast/src/type.cpp

```diff
@@ -1745,13 +1706,65 @@ size_t record_type::num_fields() const noexcept {
 }
 
 size_t record_type::num_leaves() const noexcept {
```

Comment:
I can do that, although the function is _really_ well tested because it's called indirectly from so many other places. Still makes sense to test it as a unit, though.

Reply:
Done.